app/src/util/string_utils.py

```diff
@@ -17,3 +25,71 @@ def join_list(joining_list: Optional[list], join_txt: str = "\n") -> str:
 def basic_ascii(text: str) -> str:
     # See https://www.ascii-code.com/
     return "".join([c if 32 <= ord(c) <= 126 else " " for c in text])
+
+
+# Set the nltk.data.path to a relative directory so that it's available in the Docker environment
+_nltk_data_path = os.path.abspath("./nltk_data")
+nltk.data.path.append(_nltk_data_path)
```

Comment:
Good call! I had a `nltk.data.path.clear()` before this line at one point but have since removed it. That was causing test failures elsewhere b/c another library was using nltk.